src/test/java/com/xceptance/neodymium/testclasses/data/RandomnessOfDataSets.java

```diff
@@ -0,0 +1,43 @@
+package com.xceptance.neodymium.testclasses.data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.xceptance.neodymium.NeodymiumRunner;
+import com.xceptance.neodymium.module.statement.testdata.DataSet;
+import com.xceptance.neodymium.module.statement.testdata.SuppressDataSets;
+import com.xceptance.neodymium.util.DataUtils;
+
+@RunWith(NeodymiumRunner.class)
+public class RandomnessOfDataSets
+{
+    private static List<String> datasets = new ArrayList<String>();
+
+    @DataSet(
+             randomSets = 4)
+    @Test
+    public void test1()
+    {
+        datasets.add(DataUtils.asString("key1"));
+    }
+
+    @SuppressDataSets
+    @Test
+    public void test2()
+    {
+        boolean changedOrder = false;
+        for (int i = 0; i < datasets.size(); i++)
```

Comment:
init i with 1 to avoid the addition later

Reply:
Done